### ISSUE TITLE
Config patch ignored when saving config

### ISSUE THREAD

**Issue by Entixs** [pymedusa/Medusa#8477]
**Describe the bug**
Looks like the config is being ignored when i save something on it.

**To Reproduce**
1. Go to 'Anime settings'
2. Click on 'Go to look and feel'
3. Click on 'the tab to turn on Connect anime to anime list'
4.Click on save settings.

**Expected behavior**
That the anime connects to the anime list fine but it does not. This error keeps on happening.

**Screenshots**

**Medusa (please complete the following information):**
 - OS: Linux-5.3.0-64-generic-x86_64-with-Ubuntu-19.10-eoan
 - Branch: develop
 - Commit: 56b6c356079a807889ef096cc2b5b60946ef522e
 - Python version: 	3.7.5
 - Database version: 44.16

**Debug logs (at least 50 lines):**
General > Advanced Settings > Enable debug
<details>

```
2020-09-10 14:54:44 DEBUG	APIv2-Thread-4 :: [56b6c35] Broken providers already updated in the last hour
2020-09-10 14:54:44 WARNING	APIv2-Thread-4 :: [56b6c35] Config patch ignored {'anime': {'animeSupport': False}}
```
</details>

**Additional context**

**Comment by p0psicles** [comment]
Fixed in #8448

**Comment by gniting** [comment]
I am getting similar warning when saving changes under `settings >> general` and `settings >> search settings`. The warning when I save under the `general` section is:
```
2021-04-13 09:52:28 WARNING  APIv2-Thread_3 :: [] Config patch ignored {'layout': {'historyLimit': 100}}
```
and the warning when I save under `search settings` is:
```
2021-04-13 09:54:09 WARNING  APIv2-Thread_2 :: [] Config patch ignored {'search': {'general': {'downloadHandlerFrequency': 60, 'mindownloadHandlerFrequency': 5}}}
```
